optaplanner-examples/src/test/java/org/optaplanner/examples/meetingscheduling/solver/MeetingSchedulingScoreConstraintTest.java

```diff
@@ -37,14 +38,14 @@
 public class MeetingSchedulingScoreConstraintTest {
 
     private HardMediumSoftScoreVerifier<MeetingSchedule> scoreVerifier = new HardMediumSoftScoreVerifier<>(
-            SolverFactory.createFromXmlResource("org/optaplanner/examples/meetingscheduling/solver/meetingSchedulingSolverConfig.xml"));
+            SolverFactory.createFromXmlResource(MeetingSchedulingApp.SOLVER_CONFIG));
 
     private MeetingSchedule getMeetingSchedule(int numberOfEntities) {
         // After getting the solution, need to set AttendanceList for it. And for every meeting Required & Preferred attendance lists
         MeetingSchedule solution = new MeetingSchedule();
-        MeetingParametrization parametrization = new MeetingParametrization();
+        MeetingWeightPack parametrization = new MeetingWeightPack();
```

Comment:
Shouldn't the variable name "parametrization" also be renamed? (e.g. to weightPack)